src/applications/vre/28-8832/config/chapters/claimant-information/claimantInformation.js

```diff
@@ -3,63 +3,35 @@ import merge from 'lodash/merge';
 import ssnUI from 'platform/forms-system/src/js/definitions/ssn';
 import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
 import { hasSession } from 'platform/user/profile/utilities';
+import { claimantInformation } from '../../utilities';
 
-export const schema = {
-  type: 'object',
-  properties: {
-    firstName: {
-      type: 'string',
-      maxLength: 100,
+export const schema = claimantInformation;
+
+export const uiSchema = {
+  fullName: {
+    first: {
+      'ui:title': 'First name',
+      'ui:required': () => !hasSession(),
     },
-    middleName: {
-      type: 'string',
-      maxLength: 100,
+    middle: {
+      'ui:title': 'Middle name',
     },
-    lastName: {
-      type: 'string',
-      maxLength: 100,
+    last: {
+      'ui:title': 'Last name',
+      'ui:required': () => !hasSession(),
     },
     suffix: {
-      type: 'string',
-      enum: ['Jr', 'Sr'],
-    },
-    ssn: {
-      type: 'string',
-    },
-    vaFileNumber: {
-      type: 'string',
-    },
-    dateOfBirth: {
-      pattern:
-        '^(\\d{4}|XXXX)-(0[1-9]|1[0-2]|XX)-(0[1-9]|[1-2][0-9]|3[0-1]|XX)$',
-      type: 'string',
-    },
-  },
-};
-
-export const uiSchema = {
-  firstName: {
-    'ui:title': 'First name',
-    'ui:required': () => !hasSession(),
-  },
-  middleName: {
-    'ui:title': 'Middle name',
-  },
-  lastName: {
-    'ui:title': 'Last name',
-    'ui:required': () => !hasSession(),
-  },
-  suffix: {
-    'ui:title': 'Suffix',
-    'ui:options': {
-      widgetClassNames: 'usa-input-medium',
+      'ui:title': 'Suffix',
+      'ui:options': {
+        widgetClassNames: 'usa-input-medium',
+      },
     },
   },
   ssn: {
     ...ssnUI,
     'ui:required': () => !hasSession(),
   },
-  vaFileNumber: {
+  VAFileNumber: {
```

Comment:
Should this be `VaFileNumber`? I think the backend will have an issue converting camel case to snake case with this naming.

Reply:
Good question. Not sure. Won't eventual inflection changes on the backend force `VA` for all instances of `vA`, `Va` etc? Still unclear on that whole thing.

Reply:
Eventual is an undefined amount of time 😉

Reply:
We took a look at it, and it seems like it won't be an issue when we send the payload to vets-api, so I think we're safe here.